blogs/posts/2024-08-08-map-and-nest/map-and-nest.qmd

```diff
@@ -56,7 +51,26 @@ We can grab these mini datasets in the usual way and explore them interactively.
 group_by_ics$raw_data[[1]]
 ```
 
-Next, let's apply some simple processing, say converting absolute numbers into percentages, to each of the ICSs in turn. We use `mutate()` to create a new list-column `staff_percent` and `map()` to apply the processing function to each cell in turn. ^[In this example, we actually didn't need to nest first. We could have performed the `mutate()` step on the full dataset.]
+Next, let's apply some simple processing, say converting absolute numbers into percentages, to each of the ICSs in turn.
+
+<details>
```

Comment:
I'll leave it for you to decide if you want to do like:

```
<details><summary>Click for function definition.</summary>
```

Reply:
Good shout again - thanks!